cinema_service/settings.py

```diff
@@ -9,8 +9,14 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
+
 from datetime import timedelta
 from pathlib import Path
+import os
```

Comment:
Consider using `config` from `decouple` to manage environment variables instead of `os.getenv` for consistency, as `decouple` is already imported.